vendor/broccoli-fontawesome-pack.js

```diff
@@ -46,12 +46,19 @@ FontAwesomePack.prototype.build = function() {
   if(this.options.icons === 'all'){
     selectedIcons = Object.keys(pack[pack.prefix])
   } else {
-    selectedIcons = this.options.icons
+    selectedIcons = this.options.icons.map(iconName => {
+      const prefix = 'fa';
```

Comment:
Let's grab this from the `config.familyPrefix`. That sucker is configurable.

Reply:
Done.